### PR TITLE
[objc][docs] Add iOS getting started docs.

### PR DESCRIPTION
**Pull request by rolfbjarne**

### CODE REVIEW

**Comment by spouliot** [comment]
@risi another set of eye would be welcome :) thanks!

**Comment by timrisi** [comment]
Already lookin at it :)

File: docs/getting-started-objc-ios.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+* Earlier versions of macOS, Xcode and Mono _might_ work, but are untested and unsupported;
+* Code generation can be done on Windows, but it is only possible to compile it on a Mac computer where Xcode is installed;
+
+## Hello world
+
+First let's build a simple hello world example in C#.
+
+### Create C# sample
+
+Open Visual Studio for Mac, and create a new iOS Class Library project, and name it `hello-from-csharp`, and save it to `~/Projects/hello-from-csharp`.
```
Comment (timrisi) [inline docs/getting-started-objc-ios.md]: Open Visual Studio for Mac, and create a new iOS Class Library project, name it `hello-from-csharp`, and save it to `~/Projects/hello-from-csharp`.

(don't need the first 'and')

File: docs/getting-started-objc-ios.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+		Text = "Hello from C#";
+	}
+}
+```
+
+Build the project, the resulting assembly will be saved as `~/Projects/hello-from-csharp/hello-from-csharp/bin/Debug/hello-from-csharp.dll`.
+
+### Bind the managed assembly
+
+Run the embeddinator, to create a native framework for the managed assembly:
```
Comment (timrisi) [inline docs/getting-started-objc-ios.md]: Run the embeddinator to create a native framework for the managed assembly:

File: docs/getting-started-objc-ios.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+```shell
+cd ~/Projects/hello-from-csharp
+/Library/Frameworks/Xamarin.Embeddinator-4000/Commands/objcgen ~/Projects/hello-from-csharp/hello-from-csharp/bin/Debug/hello-from-csharp.dll --target=framework --platform=iOS --outdir=output -c --debug
+```
+
+The framework will be placed in `~/Projects/hello-from-csharp/output/hello-from-csharp.framework`.
+
+### Use the generated output in an Xcode project
+
+Open Xcode, and create a new iOS Single View Application, name it `hello-from-csharp`.
```
Comment (timrisi) [inline docs/getting-started-objc-ios.md]: Open Xcode and create a new iOS Single View Application; name it `hello-from-csharp`.

File: docs/getting-started-objc-ios.md (hunk 1 above)
Comment (rolfbjarne) [inline docs/getting-started-objc-ios.md]: There were three `and`s, I removed the first two.

**Comment by rolfbjarne** [comment]
@timrisi thanks, should be fixed now 😄

**Pull request merged**